.buildkite/pipeline.yml

```diff
@@ -100,13 +49,12 @@ steps:
     artifact_paths: [Bugsnag-*.zip]
     timeout_in_minutes: 30
 
-  ##############################################################################
   #
-  # Build E2E Fixtures
+  # Build Test Fixtures
   #
-  ##############################################################################
 
-  - name: 'Build Android test fixtures - UE 4.27'
+  # UE 4.27
+  - name: ':android: Build Test Fixtures - UE 4.27'
```

Comment:
Personally I like having the platform spelled out in the step name, not just using the iOS/macOS/Android emoji.  I find it easier to read and also allows you to "Find on page" in the browser should you so wish.